tests/gbs/test_pp.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Tests for the the point process functions"""
+
+import pytest
+import numpy as np
+from strawberryfields.gbs.pp import rbfkernel
+
+pytestmark = pytest.mark.gbs
+
+
+def test_rbfkernel():
```

Comment:
How was the `Kref` matrix built? It's not really testing much if this matrix was generated from the function itself...

Reply:
It was created with numpy using a different function.

Reply:
I actually think it's ok, as long as we're certain that the hard coded example is correct - which we could check by hand for a problem of this size.

Reply:
I checked it by the Ubuntu calculator and OpenOffice calc.